rllib/evaluation/worker_set.py

```diff
@@ -98,9 +98,10 @@ def add_workers(self, num_workers: int) -> None:
         remote_args = {
             "num_cpus": self._remote_config["num_cpus_per_worker"],
             "num_gpus": self._remote_config["num_gpus_per_worker"],
-            "memory": self._remote_config["memory_per_worker"],
-            "object_store_memory": self._remote_config[
-                "object_store_memory_per_worker"],
+            # memory=0 is an error, but memory=None means no limits.
```

Comment:
@sven1977 can you double check this is ok?

Reply:
Makes sense. Thanks for fixing this.